polar-core/src/error.rs

```diff
@@ -477,6 +480,14 @@ impl fmt::Display for ValidationError {
             Self::UndefinedRule { rule_name } => {
                 write!(f, r#"Call to undefined rule "{}""#, rule_name)
             }
+            Self::MissingRequiredRule { rule } => {
+                write!(
+                    f,
+                    r#"Missing implementation for required rule {}({});"#,
+                    rule.name,
+                    format_params(&rule.params, ", ")
```

Comment:
Unless I'm missing something, it seems like we could use the existing `Display` impl:

https://github.com/osohq/oso/blob/f0b7156e2a9bcbacd314d90b391c5e9e4f8fe3e3/polar-core/src/formatting.rs#L317-L323

Reply:
yep 💯 correct. just trimmed this up.